runtime/vm/visible.c

```diff
@@ -356,7 +356,7 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if (J9_VISIBILITY_ALLOWED == result) {
 			clazz->nestHost = nestHost;
-		} else if ((J2SE_VERSION(vmThread->javaVM) >= J2SE_V15)
+		} else if ((J2SE_VERSION(vmThread->javaVM) > J2SE_V11)
```

Comment:
I don't think we should rewrite history. The condition should be `JAVA_SPEC_VERSION >= 15`.

Reply:
Agree with `JAVA_SPEC_VERSION` part, `15` is out of service, LTS version number looks consistent.

Reply:
Yes, 15 is no longer supported, but the related concepts were introduced in 15, not 12.

Reply:
If Keith is willing to accept it, I suggest `>= J2SE_V16` is appropriate.

Reply:
Any code specific to a version after a LTS version (in this case `11`)  eventually only applies to next LTS version `17` and onwards, an intermediate version just has 6 month life and the features introduced at that version becomes next LTS belongs, I think the code appears cleaner when associated with LTS versions.

Reply:
I don't understand the motivation to rewrite history. A changed that occurred due to work in Java N should not make reference to any version except N. To do so, lies to future readers of the code.

Reply:
The history is available in the repo. Intermediate version JDKs are for test/development purpose, not recommended for product usage. The JDKs in production _should_ be LTS, so the code associating with LTS version is easier to map to the product version. Hope this makes sense.

Reply:
Yes, I understand revision history is available in git. That's not the type of history I'm referring to here. When someone in the future looks at this code, we should not lie to them about when the code guarded by the test became relevant. I hope you'll agree that it's a much simpler matter to translate '15' to a supported LTS version.

Reply:
imo there is little value to keep references of those short-lived JDK versions but I can live with it.

Pls have another look.